external/mmdetection/detection_tasks/extension/datasets/mmdataset.py

```diff
@@ -193,7 +193,7 @@ def prepare_train_img(self, idx: int) -> dict:
         :param idx: int, Index of data.
         :return dict: Training data and annotation after pipeline with new keys introduced by pipeline.
         """
-        item = deepcopy(self.data_infos[idx])
+        item = copy(self.data_infos[idx])  # Copying dict(), not contents
```

Comment:
Is there a test to make sure that this change does not break mmdetection?

Reply:
Thank you for the comment. Direct test is not easy because many data preprocessing piplelines manipulates images & labels. However, almost all the train/infer tests in OTE uses the changed code block so that we could observe many test failures if there is some error.
Actually, I'm looking into some failures to figure out whether it's copy problem or caller's problem.